src/screens/ExchangeModal.js

```diff
@@ -325,6 +360,13 @@ export default function ExchangeModal({
       });
     }
 
+    const gasPrice = get(selectedGasPrice, 'txFee.native.value.amount');
```

Comment:
can get rid of lodash `get` and just do `selectedGasPrice?.txFee?.native?.value?.amount`